package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```

Comment:
- [ ] Please as the requirement say that you should have npm `start` `build` 

![Capture web_11-5-2022_103327_github com](https://user-images.githubusercontent.com/61562626/167818494-8baf7f64-0989-4d38-aa68-71390dd04480.jpeg)